src/components/book/Book.js

```diff
@@ -19,6 +20,7 @@ function Book({
           <p>ISBN: {isbn}</p>
           <p>Published {publishedDate}</p>
           <p>{totalPages} pages</p>
+          <Link to={`/books/:id`}>Books</Link>
```

Comment:
This should link to `/books` I believe since you're already looking at a specific BookDetails component. Alternatively you could just remove this <Link> since you already have a <Link> that goes to `/books` in your App.js component.